### PR TITLE
Adds a lot of inhand sprites for minerals and sheets

### PR DESCRIPTION
**Pull request by Bokkiewokkie**
## About The Pull Request

It eh, adds a ton of inhand sprites that don't exist yet

## Why It's Good For The Game

There is a great imbalance between regular metal and glass and the other minerals in this game, for there is a clear indication to everyone in the hallway that the one assistant over there is holding a few sheets of metal in their hand without them even having to inspect the fine fellow. Though that very same assistant could very well be holding uranium in their other hand without a single soul being aware of it. That is why I made this PR, it's to remove the extreme inbalance between regular sheets and the other (potentially dangerous) materials that the game has to offer.

## Changelog
:cl:
imageadd: Adds inhands for sheets and minerals.
/:cl:

If you've looked at the image diff bot and can't visualize what the sprites would look like in someone's hand let me know and I'll take a few screenshots.

### CODE REVIEW

**Comment by Kmc2000** [comment]
Nice. Can you test our durasteel and nanocarbon stuff to see if it works?

**Comment by Bokkiewokkie** [comment]
Alright I've moved the files to the nsv13 folder now.
Also here's a cool picture of the nsv materials:
image
(Do note that the corners of the durasteel are in fact rounded but the jumpsuit belt is just the same color making it look square sometimes)

File: code/game/objects/items/stacks/sheets/sheets.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /obj/item/stack/sheet
 	name = "sheet"
-	lefthand_file = 'icons/mob/inhands/misc/sheets_lefthand.dmi'
-	righthand_file = 'icons/mob/inhands/misc/sheets_righthand.dmi'
+	lefthand_file = 'nsv13/icons/mob/inhands/misc/sheets_lefthand.dmi'
+	righthand_file = 'nsv13/icons/mob/inhands/misc/sheets_righthand.dmi'
```
Comment (francinum) [inline code/game/objects/items/stacks/sheets/sheets.dm:4]:
```suggestion
	lefthand_file = 'nsv13/icons/mob/inhands/misc/sheets_lefthand.dmi' //NSV13
	righthand_file = 'nsv13/icons/mob/inhands/misc/sheets_righthand.dmi' //NSV13

```

**Comment by francinum** [comment]
@Kmc2000 Needs to be testmerged a round for final checks or otherwise confirmed.

**Comment by covertcorvid** [comment]
Seems alright, just needs a test merge

**Pull request merged**